misc/multistream-select/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.9.0 [unreleased]
 
+- Make the `V1Lazy` upgrade strategy more interoperable with `V1`.
```

Comment:
How about this:
```suggestion
- Make the `V1Lazy` upgrade strategy more interoperable with `V1`. Specifically,
  the listener now behaves identically with `V1` and `V1Lazy`. Furthermore, the
  multistream-select protocol header is now also identical, making `V1` and `V1Lazy`
  indistinguishable on the wire. The remaining central effect of `V1Lazy` is that the dialer,
  if it only supports a single protocol in a negotiation, optimistically settles on that
  protocol without immediately flushing the negotiation data (i.e. protocol proposal)
  and without waiting for the corresponding confirmation before it is able to start
  sending application data, expecting for the used protocol to be confirmed with
  the response.
```

Reply:
Yes, perfect. I think that is very helpful. 

In "expecting for the used protocol to be confirmed with the response." I am not sure the "for" is needed, but I might just be mistaken.